flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSink.java

```diff
@@ -123,6 +123,8 @@ public class HiveTableSink implements DynamicTableSink, SupportsPartitioning, Su
 
     private static final Logger LOG = LoggerFactory.getLogger(HiveTableSink.class);
 
+    public static final String BATCH_COMPACT_WRITER_OP_NAME = "batch_compact_writer";
```

Comment:
Maybe better to change to `batch_writer` since stream mode is `stream_writer`
```suggestion
    public static final String BATCH_COMPACT_WRITER_OP_NAME = "batch_writer";
```